### DISCUSSION TITLE
aria2/1.34.0 是哪家的离线下载服务器么

### DISCUSSION THREAD

**Discussion by qwqcwcawa** [c0re100/qBittorrent-Enhanced-Edition#544] [General]
![Screenshot_2024-04-21-19-12-56-782_me fengmlo qbRemoteFree-edit](https://github.com/c0re100/qBittorrent-Enhanced-Edition/assets/156668576/4da545ce-328d-4e3a-a876-d335227fed09)
![IMG_20240421_204131](https://github.com/c0re100/qBittorrent-Enhanced-Edition/assets/156668576/0f5e1f33-2749-4a82-b203-58fe734f319e)
![Screenshot_2024-04-21-20-41-53-088_me fengmlo qbRemoteFree-edit](https://github.com/c0re100/qBittorrent-Enhanced-Edition/assets/156668576/130d12cc-1e32-4fb9-8e8b-2381a030b046)
![IMG_20240421_204502](https://github.com/c0re100/qBittorrent-Enhanced-Edition/assets/156668576/8feac252-3789-4724-936f-30db1c4a2cc2)
![Screenshot_2024-04-18-23-55-50-626_dev bartuzen qbitcontroller-edit](https://github.com/c0re100/qBittorrent-Enhanced-Edition/assets/156668576/a4c3a921-91ca-4d2c-8dfc-184d61e3e61a)
有时候会看到列表里有116.132开头使用aria2/1.34.0的(有的进度和上传的数据量不符)
从以前的记录中可以看到116.132有不少使用libtorrent 2.0.7被标记为offline downloader的ip
[banned_peers.csv](https://github.com/c0re100/qBittorrent-Enhanced-Edition/files/15051975/banned_peers.csv)

**Comment by qpzr** [comment]
看了一眼 IP过滤文件，上个月我也整段拉黑过 116.132.217.0-116.132.217.255  也是这个客户端和ID
半夜我下刚发布的电视剧，他就疯狂的吸一点上传都不给我，下完就跑的

**Reply by qwqcwcawa** [reply]
123.182.48.0/24也有这种客户端，直接屏蔽aria2/1.34.0的话应该会有不少误伤

**Reply by qpzr** [reply]
挂着aria2的名头，各种伪装ID ，有UT的 有TR的 有LT的 有lt的。斗智斗勇不如丢给迅雷离线，觉得心累，直接送aria2进黑名单。
